style.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #f7f7f9;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  height: 48px;
+}
+
+header a {
+  font-family: "Poppins", Helvetica, sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  color: #6070ff;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+}
+
+.headline {
+  display: flex;
```

Comment:
**OPTIONAL**
- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)